dev/releases/utils.py

```diff
@@ -152,6 +152,38 @@ def check_whether_github_release_exists(tag):
             return True
     return False
 
+# lightweight vs annotated
+# https://stackoverflow.com/questions/40479712/how-can-i-tell-if-a-given-git-tag-is-annotated-or-lightweight#40499437
+def is_annotated_git_tag(tag):
+    is_annotated = subprocess.run(["git", "for-each-ref", "refs/tags/" + tag],
+                                  check=True, capture_output=True, text=True)
+    if "tag" == is_annotated.stdout.split()[1]:
+        return True
+    if "commit" == is_annotated.stdout.split()[1]:
+        return False
+    error(f"could not check whether tag {tag} is annotated")
+
+def get_commit_of_git_tag(tag):
+    result = subprocess.run(["git", "show-ref", "--tags", "--dereference", tag],
+                            check=True, capture_output=True, text=True)
+    result = result.stdout.split('\n')
+    # TODO finish this
+    return commit
+
+def check_git_tag_for_release(tag):
+    if not does_git_tag_exist(tag):
+        error(f"There exists no tag {TAG_NAME}")
```

Comment:
```suggestion
    if not does_git_tag_exist(tag):
        error(f"There exists no tag {tag}")
```
but I'd rather use the "improved" `is_annotated_git_tag` I suggest above:
```suggestion
    if not is_annotated_git_tag(tag):
        error(f"There is no annotated tag {tag}")
```